packages/core/next.config.js

```diff
@@ -24,7 +24,12 @@ const nextConfig = {
   // TODO: We won't need to enable this experimental feature when migrating to Next.js 13
   experimental: {
     scrollRestoration: true,
-    // Trace user's `node_modules` dir
+    /*
+     * The FastStore Discovery CLI will update this value to match the path where the
+     * command is being run, because that is where the node_modules directory is.
+     * For discovery-only paths, that the user directory, for monorepo, that the base
+     * of the monorepo
```

Comment:
question: Idk if I understood correctly, is it saying what this `outputFileTracingRoot` should be on those cases?
```suggestion
     * For discovery-only paths, that is the user directory, and for monorepo, that is the base
     * of the monorepo
```

Reply:
You're right. I don't know what the hell happened with the text. I thought I had written it correctly 😆 Thank you for the suggestion!